src/transaction/hooks/useGetTransactionStatus.tsx

```diff
@@ -28,8 +28,10 @@ export function useGetTransactionStatus() {
       //   </a>
```

Comment:
@abcrane123 can you remind me what's the story with this?

Reply:
`useWriteContracts` returns a transaction id that looks something like `0xf595f3f5111f88b30c668946d4430cf74326aadec59fe846…0000000000000000000000000000000000000000000014a34`. i asked connor how i can use this id to get the correct block explorer link for a pending transaction. i pass this id to `useCallsStatus` to get the txn hash which i use to get the "view transaction" block explorer link but from the designs it looks like there is a way to get the explorer link with that transaction id.